src/reducers/PlaylistReducer.js

```diff
@@ -0,0 +1,25 @@
+export const PlaylistReducer = (playlistState, { type, payload }) => {
```

Comment:
liked the way how you are destructuring it, this would have reduced lot of boilerplate code